resources/assets/js/components/Book/BooksCommentsComponent.vue

```diff
@@ -14,22 +22,32 @@
 
 <script>
     export default {
-        props: ['book', 'new_comment_route', 'comments'],
+        props: ['book', 'new_comment_route', 'book_comments', 'user'],
```

Comment:
Can we do this? and directly use `comments` variable at all places? instead of having two variables `all_comments` and `book_comments`
```suggestion
        props: ['book', 'new_comment_route', 'comments', 'user'],
```

Reply:
also I'm not sure. Are we using camelCasing or snake_casing?

Reply:
I am not able to send the props as camel case in vue js. 

Do you have any idea ?